apple/RNGestureHandlerModule.mm

```diff
@@ -99,10 +99,17 @@ void decorateRuntime(jsi::Runtime &runtime)
         if (!arguments[0].isObject()) {
           return jsi::Value::null();
         }
-        auto shadowNode = arguments[0].asObject(runtime).getNativeState<ShadowNode>(runtime);
-        bool isFormsStackingContext = shadowNode->getTraits().check(ShadowNodeTraits::FormsStackingContext);
 
-        return jsi::Value(isFormsStackingContext);
+        auto arg = arguments[0].asObject(runtime);
+
+        if (arg.hasNativeState<ShadowNodeWrapper>(runtime)) {
+          auto shadowNodeWrapper = arg.getNativeState<ShadowNodeWrapper>(runtime);
+          bool isFormsStackingContext =
+              shadowNodeWrapper->shadowNode->getTraits().check(ShadowNodeTraits::FormsStackingContext);
+          return jsi::Value(isFormsStackingContext);
+        }
+
+        return jsi::Value(true);
```

Comment:
Can we unify the implementation here with `cpp-adapter.cpp`?

Reply:
Done in [2e06198](https://github.com/software-mansion/react-native-gesture-handler/pull/2966/commits/2e0619802bb3db99a79f2ee7252b8374032b373c). Thanks for catching this!